lib/iupac.rb

```diff
@@ -3,22 +3,26 @@
 module Iupac_convention
 
   Suffix =                    #   This is a constant definition
-      {                     
+      {                       # This is a hash (i.e. associative array  where a number of 'keys' are associated with corresponding 'values')
     :alkane => "ane",
      :alcohol => "ol",
      
   }
   
   
```

Comment:
If I use  "compound_name.split(/ /)" this would separated each word, bracket, hyphen and space. How do i then put this into an array of strings again ?